com.chartboost.mediation/Documentation/integration/loading-ads.md

```diff
@@ -99,28 +99,51 @@ Banners are now shown automatically after load, as such you will need to pass a
 | `ChartboostMediationBannerAdScreenLocation.BottomCenter` | 5          | Positions the banner to the bottom-center screen of the device. |
 | `ChartboostMediationBannerAdScreenLocation.BottomRight`  | 6          | Positions the banner to the bottom-right screen of the device.  |
 
-If you enable auto-refresh for a banner placement in the dashboard, then the Chartboost Mediation Unity SDK will apply that setting when the placement is shown.
 
-> **Note** \
-> Any auto refresh changes made on the dashboard will take approximately one hour to take effect and the SDK must be rebooted in order to pick up the changes once they are available.
+If you want to place the bannerView using a custom screen location then you can do so by providing a screen co-ordinate(x, y) which denotes where the top-left corner of this bannerView will be placed
+```C#
+// 400 pixels from left
+float x = ChartboostMediationConverters.PixelsToNative(400); 
+// 200 pixels from bottom
+float y = ChartboostMediationConverters.PixelsToNative(200); 
+
+// Load the banner ad
+await bannerView.Load(loadRequest, x, y);
+```
 
-You will need to create an instance for each Placement Name you want to use. Finally, make the call to load the ad:
+### Using `ChartboostMediationUnityBannerAd` API
+
+`ChartboostMediationUnityBannerAd` API enables loading of a bannerAd within a unity gameobject. 
+To create such gameobject you can simply right-click in hierarchy window and select `Chartboost Mediation/UnityBannerAd`
```

Comment:
To create this gameobject, right-click in the hierarchy window and select `Chartboost Mediation/UnityBannerAd`